cetmix_tower_server/models/cx_tower_plan_line.py

```diff
@@ -25,6 +25,19 @@ class CxTowerPlanLine(models.Model):
         auto_join=True,
         ondelete="cascade",
     )
+    related_plan_line_ids = fields.One2many(
```

Comment:
This field can be related. `related = "plan_id.line_ids"`. And use the same name as the destination related field.